interfaces/Types.ts

```diff
@@ -47,6 +48,7 @@ export const PIPELINE_TYPE_PRETTY_NAMES: { [key in PipelineType]: string } = {
 	/// audio
 	"text-to-speech":                                           "Text-to-Speech",
 	"automatic-speech-recognition":                             "Automatic Speech Recognition",
+	"audio-to-audio":                                           "Audio to Audio(s)",
```

Comment:
```suggestion
	"audio-to-audio":                                           "Audio-to-Audio",
```

looks better like that, no?

Reply:
Yes !